nejm-catalyst.csl

```diff
@@ -0,0 +1,187 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="in-text" version="1.0" demote-non-dropping-particle="sort-only" page-range-format="minimal" default-locale="en-US">
+  <info>
+    <title>NEJM Catalyst</title>
+    <title-short>NEJM Catalyst</title-short>
+    <id>http://www.zotero.org/styles/nejm-catalyst</id>
+    <link href="http://www.zotero.org/styles/nejm-catalyst" rel="self"/>
+    <link href="https://catalyst.nejm.org/pb-assets/images/NEJM%20Catalyst%20Reference%20style-1589400257323.pdf" rel="documentation"/>
```

Comment:
@badgettrg, thanks. Where did you find this PDF link? I only found links to https://catalyst.nejm.org/pb-assets/author-center/ReferenceswLinks_Jul2020-1595613582897.pdf at https://catalyst.nejm.org/author-center/article-types, although that seems to describe the same style.